bentoml/deployment/clipper/__init__.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2019 Atalaya Tech, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import shutil
+
+import docker
+
+from bentoml.archive import load
+from bentoml.handlers import ImageHandler
+from bentoml.deployment.base_deployment import Deployment
+from bentoml.deployment.utils import generate_bentoml_deployment_snapshot_path, \
+    process_docker_api_line
+from bentoml.deployment.clipper.templates import DEFAULT_CLIPPER_ENTRY
+from bentoml.utils.exceptions import BentoMLException
+
+
+def build_docker_image(snapshot_path):
+    docker_api = docker.APIClient()
+    tag = "something"
+    for line in docker_api.build(path=snapshot_path, dockerfile='Dockerfile-clipper', tag=tag):
+        process_docker_api_line(line)
+    return tag
+
+
+class ClipperDeployment(Deployment):
+    """Clipper deployment for BentoML bundle
+    """
+
+    def __init__(self, clipper_conn, archive_path, api_name, slo_micros=100000):
```

Comment:
remove slo_micros since we will not handle register application

Reply:
done